rawrequest/client.go

```diff
@@ -7,12 +7,24 @@ import (
 	stripe "github.com/stripe/stripe-go/v80"
 )
 
+// Client is used to invoke /quotes APIs.
```

Comment:
quotes API?

Reply:
thanks for catching this; it was a typo.  fixing!